d2/src/main/java/com/linkedin/d2/balancer/servers/ZooKeeperAnnouncer.java

```diff
@@ -162,11 +172,10 @@ public synchronized void markUp(final Callback<None> callback)
 
   private synchronized void doMarkUp(Callback<None> callback)
   {
-    _server.markUp(_cluster, _uri, _partitionDataMap, _uriSpecificProperties, new Callback<None>()
-    {
+
+    final Callback<None> markUpCallback = new Callback<None>() {
```

Comment:
Looks like the style in this file is to have the opening curly brace on a separate line.  It would be good to match the existing style & move your curly brace down onto the next line.

Reply:
Thank you for pointing this out. Correcting in the next version of the PR.